doc/Glossary/index.md

```diff
@@ -5,74 +5,198 @@ permalink: /:path/
-| Transport channel                                       | A Kaa client component responsible for communication with a Kaa server. Refer to Transports for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-| U                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| User verifier                                           | A server component that handles the user verification. Refer to User verifiers for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
```

Comment:
A predicate expression based on the Spring Expression Language designed to filter endpoint profiles.
By applying profile filters Kaa platform identifies which endpoints should be attributed to a specific endpoint group.